docs/getting-started.md

```diff
@@ -109,3 +109,62 @@ You can now continue using your UP address within the dApp:
 ```javascript
 const myUPAddress = myContracts.ERC725Account.address;
 ```
+
+## Proxy Deployment
+
+LSPFactory uses proxy deployment of smart contracts to maximise gas efficiency. This can be configured inside the `options` object when deploying [Universal Profiles](./deployment/universal-profile.md) or [Digital Assets](./deployment/digital-asset.md).
```

Comment:
Suggestion changes:

**before:**
LSPFactory uses proxy deployment of smart contracts to maximise gas efficiency 

**after:**
LSPFactory deploys smart contracts as proxies to reduce gas deployment costs.